src/custom/components/TransactionConfirmationModal/index.tsx

```diff
@@ -480,48 +191,113 @@ export function ConfirmationPendingContent({
-        <h3>
-          <span>{operationMessage} </span>
-          <span>
-            <Trans>Follow these steps:</Trans>
-          </span>
-        </h3>
-
-        <StepsWrapper>
-          <div>
-            <StepsIconWrapper>
-              <UserCheck />
-            </StepsIconWrapper>
-            <p>
-              <Trans>
-                Sign the {operationLabel} with your {walletNameLabel}{' '}
-                {account && <span>({ensName || shortenAddress(account)})</span>}
-              </Trans>
-            </p>
-          </div>
-          <hr />
-          <div>
-            <StepsIconWrapper>
-              <CheckCircle />
-            </StepsIconWrapper>
-            <p>{operationSubmittedMessage}</p>
-          </div>
-        </StepsWrapper>
-      </LowerSection>
+      {/* Only shown for APPROVE_TOKEN operation for MetaMask Desktop */}
+      {isApproveMetaMaskDesktop && (
+        <ApproveWrapper>
+          <h3>
+            Review and select the ideal <br /> {compareCapLabel} in your wallet
+          </h3>
+          <ApproveComparison>
+            <CompareItem>
+              <h5>'{compareTitleLeft}'</h5>
+              <ItemList listIconAlert>
+                <li>
+                  <SVG src={alertImage} /> Approval on each order
+                </li>
+                <li>
+                  <SVG src={alertImage} /> Pay gas on every trade
+                </li>
+              </ItemList>
+            </CompareItem>
+            <CompareItem highlight recommended>
+              <h5>'{compareTitleRight}'</h5>
+              <ItemList>
+                <li>
+                  <SVG src={checkImage} /> Only approve once
+                </li>
+                <li>
+                  <SVG src={checkImage} /> Save on future gas fees
+                </li>
+              </ItemList>
+            </CompareItem>
+          </ApproveComparison>
+
+          <ApproveFooter>
+            <h6>No matter your choice, enjoy these benefits:</h6>
+            <ul>
+              <li>
+                <SVG src={checkImage} /> The contract only withdraws funds for signed open orders
+              </li>
+              <li>
+                <SVG src={checkImage} /> Immutable contract with multiple&nbsp;
+                <ExternalLink
+                  href={'https://github.com/cowprotocol/ethflowcontract/tree/main/audits'}
+                  target={'_blank'}
+                  rel={'noopener'}
+                >
+                  audits
+                </ExternalLink>
+              </li>
+              <li>
+                <SVG src={checkImage} /> Over 2 years of successful trading with billions in volume
+              </li>
+              <li>
+                <SVG src={checkImage} /> Adjust your spending cap anytime
+              </li>
+            </ul>
+          </ApproveFooter>
+        </ApproveWrapper>
+      )}
+
+      {/* Not shown for APPROVE_TOKEN operation */}
+      {!isApproveMetaMaskDesktop && (
```

Comment:
Okay, you included this. Nice!

After seeing the message i was almost fine leaving it for all wallets (works decently well even for not MM wallets). But Since you did it, maybe is best like this